partner_ref_views/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Partner Ref Views",
+    "summary": """
+        Adds a search field by Reference
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "17.0.1.0.0",
+    "category": "Extra Tools",
+    "website": "https://github.com/solvosci/slv-partner",
+    "depends": ["contacts"],
```

Comment:
This addon actually depends on `base` (as selected views in inheritance shows us), `contacts` is not required to be installed.
Then, remove depends key in this case (when we only depends on `base`, we usually can omit it)